src/generator-add-action-api-mesh.js

```diff
@@ -19,14 +19,14 @@ class ApiMeshActionGenerator extends ActionGenerator {
     this.props = {
       description: 'This is a sample action showcasing how to query API MESH',
       // eslint-disable-next-line quotes
-      requiredParams: `['MESH_ID', 'MESH_API_KEY']`,
+      requiredParams: `['MESH_ID']`,
       // eslint-disable-next-line quotes
       requiredHeaders: `['Authorization']`,
       // eslint-disable-next-line quotes
       importCode: `const { GraphQLClient, gql } = require('graphql-request')
 const { Core } = require('@adobe/aio-sdk')`,
 
-      responseCode: `const apiMeshEndpoint = \`https://graph.adobe.io/api/\${params.MESH_ID}/graphql?api_key=\${params.MESH_API_KEY}\`
+      responseCode: `const apiMeshEndpoint = \`https://edge-stage-graph.adobe.io/api/\${params.MESH_ID}/graphql\`
```

Comment:
Even though this is a sample action, most users will be using the production endpoint
```suggestion
      responseCode: `const apiMeshEndpoint = \`https://edge-graph.adobe.io/api/\${params.MESH_ID}/graphql\`
```